library/Cube/Cube.php

```diff
@@ -4,11 +4,19 @@
 
 namespace Icinga\Module\Cube;
 
+use Icinga\Application\Modules\Module;
 use Icinga\Exception\IcingaException;
+use Icinga\Module\Cube\ProvidedHook\Icingadb\IcingadbSupport;
 use Icinga\Web\View;
 
 abstract class Cube
 {
+    /** @var string Prefix for slice params */
+    public const SLICE_PREFIX = 'slice.';
```

Comment:
If I read the code correctly `slice.` is only used if the backend is Icinga DB. Instead of having multiple `ifs` for that case, please set the prefix once if the backend is Icinga DB.